### PR TITLE
Add "Enter voucher" button

### PR DESCRIPTION
**Pull request by TamoshaytisV**
https://liv-it.atlassian.net/browse/WEB-840

Added button that redirects authenticated student to page with voucher form

@polesye @leha2000 please review

### CODE REVIEW

File: templates/navigation.html [hunk 1]
```diff
@@ -68,6 +68,11 @@ <h2 class="course-header"><span class="provider">${course.display_org_with_defau
             <a href="${marketing_link('COURSES')}">${_('Browse Labs')}</a>
           </li>
         % endif
+        % if settings.LABSTER_ENABLE_VOUCHERS:
```
Comment (polesye) [inline templates/navigation.html]: please use `settings.LABSTER_FEATURES.get('ENABLE_VOUCHERS')`

**Comment by polesye** [comment]
1 comment, 👍  after fixing it.

**Comment by TamoshaytisV** [comment]
@leha2000 review please

**Comment by leha2000** [comment]
:+1:

**Pull request merged**